### PR TITLE
CASSANDRA-17955 

### PR DESCRIPTION
**Pull request by smiklosovic**
Thanks for sending a pull request! Here are some tips if you're new here:
 
 * Ensure you have added or run the [appropriate tests](https://cassandra.apache.org/_/development/testing.html) for your PR.
 * Be sure to keep the PR description updated to reflect all changes.
 * Write your PR title to summarize what this PR proposes.
 * If possible, provide a concise example to reproduce the issue for a faster review.
 * Read our [contributor guidelines](https://cassandra.apache.org/_/development/index.html)
 * If you're making a documentation change, see our [guide to documentation contribution](https://cassandra.apache.org/_/development/documentation.html)
 
Commit messages should follow the following format:

```
<One sentence description, usually Jira title or CHANGES.txt summary>

<Optional lengthier description (context on patch)>

patch by <Authors>; reviewed by <Reviewers> for CASSANDRA-#####

Co-authored-by: Name1 <email1>
Co-authored-by: Name2 <email2>

```

The [Cassandra Jira](https://issues.apache.org/jira/projects/CASSANDRA/issues/)

### CODE REVIEW

File: src/java/org/apache/cassandra/service/ActiveRepairService.java [hunk 1]
```diff
@@ -198,10 +198,10 @@ public static ThreadPoolExecutor repairCommandExecutor()
     private final Gossiper gossiper;
     private final Cache<Integer, Pair<ParentRepairStatus, List<String>>> repairStatusByCmd;
 
-    private final DebuggableThreadPoolExecutor clearSnapshotExecutor = DebuggableThreadPoolExecutor.createWithMaximumPoolSize("RepairClearSnapshot",
```
Comment (smiklosovic) [inline src/java/org/apache/cassandra/service/ActiveRepairService.java:201]: I renamed this executor as it is not only clearing snapshots anymore. Also I had to make it public to call it from outside, obviously.

Comment (dcapwell) [reply src/java/org/apache/cassandra/service/ActiveRepairService.java:201]: given this is a `DebuggableThreadPoolExecutor` the name would have been exposed via JMX, so renaming the string could be a regression if we released that name.

Comment (smiklosovic) [reply src/java/org/apache/cassandra/service/ActiveRepairService.java:201]: @dcapwell where are they exposed? Are not only `JMXEnabledThreadPoolExecutor`s exposed to a user?

Comment (dcapwell) [reply src/java/org/apache/cassandra/service/ActiveRepairService.java:201]: yes you are right, `JMXEnabledThreadPoolExecutor` exposes but this doesn't.

**Pull request closed without merging**